CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+
+## Contributing
+
+We welcome all contributions!  That said, this is a Sage Bionetworks owned project, and we use JIRA ([AG](https://sagebionetworks.jira.com/jira/software/c/projects/AG/boards/91)/[IBCDPE](https://sagebionetworks.jira.com/jira/software/c/projects/IBCDPE/boards/189)) to track any bug/feature requests. This guide will be more focussed on a Sage Bio employee's development workflow.  If you are a Sage Bio employee, make sure to assign yourself the JIRA ticket if you decide to work on it.
+
+## Coding Style
+
+The code in this package is also automatically formatted by `black` for consistency.
+
+## The Development Life Cycle
+
+<!--
+### Fork and clone this repository
+
+1. See the [Github docs](https://help.github.com/articles/fork-a-repo/) for how to make a copy (a fork) of a repository to your own Github account.
+1. Then, [clone the repository](https://help.github.com/articles/cloning-a-repository/) to your local machine so you can begin making changes.
+1. Add this repository as an [upstream remote](https://help.github.com/en/articles/configuring-a-remote-for-a-fork) on your local git repository so that you are able to fetch the latest commits.
+1. On your local machine make sure you have the latest version of the `develop` branch:
+
+    ```shell
+    git checkout develop
+    git pull upstream develop
+    ``` -->
+
+### Install development dependencies
+
+Please follow the [README.md](README.md) to install the package for development purposes.
+
+### Developing at Sage Bio
+
+The agora-data-tools project follows the standard [trunk based development](https://trunkbaseddevelopment.com/) development strategy.
+> To ensure the most fluid development, do not push to your `dev`
```

Comment:
Since we've removed instructions on forking, I think this should just be "do not push to `dev`" (without 'your').